build.sbt

```diff
@@ -1,7 +1,36 @@
-name := "sbt-dcos"
+import com.mesosphere.sbt.BuildPlugin.publishSettings
 
-version := "0.1.0-SNAPSHOT"
-
-organization := "com.mesosphere"
-
-sbtPlugin := true
+lazy val root = project.in(file("."))
+  .settings(
+    name := "sbt-dcos",
+    version := "0.1.0-SNAPSHOT",
+    organization := "com.mesosphere",
```

Comment:
A slight annoyance is that Cosmos uses `com.mesosphere.cosmos` as its `organization`. That seems incorrect for this plugin. Should we change Cosmos at some point to have `com.mesosphere` as its `organization`, for consistency?

Alternatively, should we use `io.dcos`?